_rules/SC1-2-3-video-alternative.md

```diff
@@ -0,0 +1,182 @@
+---
+name: video with audio has an accessible alternative
+rule_type: composite
+description: |
+  This rule checks video elements with audio have an alternative available as audio or as text
```

Comment:
"alternative available" -> "alternative for the video content"

Reply:
Done. Thanks.